templates/nginx-sample.conf

```diff
@@ -37,6 +37,7 @@ http {
 			proxy_http_version 1.1;
 			proxy_set_header Upgrade $http_upgrade;
 			proxy_set_header Connection "upgrade";
+			proxy_set_header Referrer-Policy "no-referrer";
```

Comment:
This is setting the header on the proxied request to the dashboard web server, not the response to the browser.

I think this should be handled in the dashboard container since it's directly related to that.